build.gradle

```diff
@@ -558,7 +558,7 @@ configure(dexerProjects) { subproject ->
     }
 
     afterEvaluate {
```

Comment:
As said depending on jar should be enough we don't want to execute tests for instance. You need to fix the classpath we pass to dexer. Try `gr checkDexer --dry-run` I guess everything is already correct